courses/beginner/10.md

```diff
@@ -48,207 +41,183 @@ control 'nginx-modules' do
 
+::: code-tabs
+@tab my_nginx_overlay
 ```ruby
-# my_nginx_overlay
-control 'first-overlay-control'
-control 'second-overlay-control'
+include_controls 'my_nginx'
 ```
-In the above example, all controls from `my_nginx` profile and `my_nginx_overlay` profile will be executed every time my-app-profile is executed **except** for control `second-overlay-control` from the `my_nginx_overlay` profile.
+:::
 
-### Modifying a Control  
+In the example above, every time `my_nginx_overlay` profile is executed, all the controls from `my_nginx` profile are also executed. Therefore, the following controls would be executed for `my_nginx_overlay`:
 
-Let’s say a particular control from an included profile should still run, but the impact isn’t appropriate? Perhaps the test should still run, but if it fails it should be treated as low severity instead of high severity?
+| Controls      |     Executed  |
+| ------------- | ------------- |
+| nginx-version |      &check;  |
+| nginx-modules |      &check;  |
+| nginx-conf-file |     &check;  |
+| nginx-conf-perms |    &check;  |
+| nginx-shell-access |  &check;  |
 
-When a control is included, it can also be modified!
+### Skipping a Control
 
+What if one of the controls from the included profile does not apply to your environment? Luckily, it is not necessary to maintain a slightly-modified copy of the included profile just to delete a control. The skip_control command tells InSpec to not run a particular control.
+
+::: code-tabs
+@tab my_nginx_overlay
 ```ruby
-# my_nginx profile
-control 'nginx-version' do
-  impact 1.0
-  title 'NGINX version'
-  desc 'The required version of NGINX should be installed.'
+include_controls 'my_nginx' do
+  skip_control 'nginx-conf-perms'
 end
+```
+:::
 
-control 'nginx-modules' do
-  impact 1.0
-  title 'NGINX modules'
-  desc 'The required NGINX modules should be installed.'
-end
+In the above example, all controls from `my_nginx` profile will be executed, **except** for control `nginx-conf-perms`, every time `my_nginx_overlay` is executed. Therefore, the following controls will be executed for `my_nginx_overlay`:
 
-control 'nginx-conf' do
-  impact 1.0
-  title 'NGINX configuration'
-  desc 'The NGINX config file should owned by root, be writable only by owner, and not writeable or and readable by others.'
-end
+| Controls      |     Executed  |
+| ------------- | ------------- |
+| nginx-version |      &check;  |
+| nginx-modules |      &check;  |
+| nginx-conf-file |     &check;  |
+| nginx-conf-perms |   &cross;   |
+| nginx-shell-access |  &check;  |
 
-include_controls 'my_nginx_overlay' do
-  control 'second-overlay-control' do
-    impact 0.5
-  end
-end
-```
+### Selectively Including Controls
+
+If there are only a handful of controls that should be executed from an included profile, it’s not necessary to skip all the unneeded controls, or worse, copy/paste those controls bit-for-bit into your profile. Instead, use the `require_controls` command.
```

Comment:
We don't have to add this today, but I think "or worse, copy/paste those controls bit-for-bit into your profile" would be worth a :::details::: or :::info::: section or something on this page somewhere talking about, Why not copy and paste? Or something like that.

Reply:
I implemented it as a footnote for now. Thanks Emily!